src/server/scripts/Spells/spell_pet.cpp

```diff
@@ -1697,67 +1697,53 @@ class spell_dk_pet_scaling_03 : public SpellScriptLoader
     }
 };
 
-class spell_dk_rune_weapon_scaling_02 : public SpellScriptLoader
+class spell_dk_rune_weapon_scaling_02 : public AuraScript
```

Comment:
DANGER

![ElectricGregariousCivet-small](https://user-images.githubusercontent.com/4329612/56523700-2a1b6580-6548-11e9-8498-7d6347f66926.gif)
(lets see who knows this **amazing** song)

IIRC this spell is also used on dk's ghoul

Reply:
Nice :rofl: Didn't notice that...

Reply:
But that's AuraScript anyway.

Reply:
@ccrs
Fire in the disco!